WalletWasabi.Fluent/Models/UI/QrCodeReader.cs

```diff
@@ -57,6 +60,8 @@ public IObservable<(string decoded, Bitmap bitmap)> Read()
 	private string Decode(PixelBufferScope scope)
 	{
 		using var bitmap = SKBitmap.Decode(scope.Buffer.ReferImage());
-		return _decoder.DecodeBitmap(bitmap)?.Text ?? "";
+		var source = new SKBitmapLuminanceSource(bitmap);
+    	var binary = new BinaryBitmap(new HybridBinarizer(source));
```

Comment:
```suggestion
		var binary = new BinaryBitmap(new HybridBinarizer(source));
```

Reply:
Thank you! Perhaps, the IDE of my choice doesn't support formatting on save, have to check that.